[2023-07-17 16:23:13,437] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 16:23:13,454] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 16:23:13,454] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 16:23:13,455] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-07-17 16:23:13,455] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 16:23:13,485] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): AAPL> on 2023-05-30 00:00:00+00:00
[2023-07-17 16:23:13,498] {standard_task_runner.py:52} INFO - Started process 9369 to run task
[2023-07-17 16:23:13,534] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'AAPL', 'scheduled__2023-05-30T00:00:00+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmplisqz9us', '--error-file', '/tmp/tmp5tu44iyv']
[2023-07-17 16:23:13,545] {standard_task_runner.py:80} INFO - Job 141: Subtask AAPL
[2023-07-17 16:23:13,744] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [running]> on host lucas-Estudos
[2023-07-17 16:23:14,131] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=AAPL
AIRFLOW_CTX_EXECUTION_DATE=2023-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-30T00:00:00+00:00
[2023-07-17 16:23:14,823] {base.py:297} ERROR - AAPL: No price data found, symbol may be delisted (1h 2023-05-29 -> 2023-05-30)
[2023-07-17 16:23:14,826] {python.py:173} INFO - Done. Returned value was: None
[2023-07-17 16:23:15,021] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=AAPL, execution_date=20230530T000000, start_date=20230717T192313, end_date=20230717T192315
[2023-07-17 16:23:15,161] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-17 16:23:15,416] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-17 16:54:28,857] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 16:54:28,882] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 16:54:28,884] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 16:54:28,884] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-07-17 16:54:28,884] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 16:54:28,924] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): AAPL> on 2023-05-30 00:00:00+00:00
[2023-07-17 16:54:28,974] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'AAPL', 'scheduled__2023-05-30T00:00:00+00:00', '--job-id', '347', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmparhmz9f3', '--error-file', '/tmp/tmpgg048eb0']
[2023-07-17 16:54:28,978] {standard_task_runner.py:80} INFO - Job 347: Subtask AAPL
[2023-07-17 16:54:28,980] {standard_task_runner.py:52} INFO - Started process 21813 to run task
[2023-07-17 16:54:29,153] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [running]> on host lucas-Estudos
[2023-07-17 16:54:29,367] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=AAPL
AIRFLOW_CTX_EXECUTION_DATE=2023-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-30T00:00:00+00:00
[2023-07-17 16:54:29,979] {base.py:297} ERROR - AAPL: No price data found, symbol may be delisted (1h 2023-05-29 -> 2023-05-30)
[2023-07-17 16:54:29,982] {python.py:173} INFO - Done. Returned value was: None
[2023-07-17 16:54:30,011] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=AAPL, execution_date=20230530T000000, start_date=20230717T195428, end_date=20230717T195430
[2023-07-17 16:54:30,037] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-17 16:54:30,107] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-17 21:01:21,248] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 21:01:21,256] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 21:01:21,256] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 21:01:21,257] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-07-17 21:01:21,257] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 21:01:21,268] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): AAPL> on 2023-05-30 00:00:00+00:00
[2023-07-17 21:01:21,272] {standard_task_runner.py:52} INFO - Started process 22016 to run task
[2023-07-17 21:01:21,275] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'AAPL', 'scheduled__2023-05-30T00:00:00+00:00', '--job-id', '568', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmpqz7aikfp', '--error-file', '/tmp/tmpur3qve3a']
[2023-07-17 21:01:21,276] {standard_task_runner.py:80} INFO - Job 568: Subtask AAPL
[2023-07-17 21:01:21,427] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [running]> on host lucas-Estudos
[2023-07-17 21:01:21,661] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=AAPL
AIRFLOW_CTX_EXECUTION_DATE=2023-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-30T00:00:00+00:00
[2023-07-17 21:01:26,596] {base.py:297} ERROR - AAPL: No price data found, symbol may be delisted (1h 2023-05-29 -> 2023-05-30)
[2023-07-17 21:01:26,603] {python.py:173} INFO - Done. Returned value was: None
[2023-07-17 21:01:26,655] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=AAPL, execution_date=20230530T000000, start_date=20230718T000121, end_date=20230718T000126
[2023-07-17 21:01:26,702] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-17 21:01:26,801] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-17 22:03:38,087] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 22:03:38,126] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-17 22:03:38,127] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 22:03:38,127] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-07-17 22:03:38,127] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-07-17 22:03:38,156] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): AAPL> on 2023-05-30 00:00:00+00:00
[2023-07-17 22:03:38,167] {standard_task_runner.py:52} INFO - Started process 65404 to run task
[2023-07-17 22:03:38,174] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'AAPL', 'scheduled__2023-05-30T00:00:00+00:00', '--job-id', '877', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmpepwabi8b', '--error-file', '/tmp/tmp5u7nj79p']
[2023-07-17 22:03:38,175] {standard_task_runner.py:80} INFO - Job 877: Subtask AAPL
[2023-07-17 22:03:38,252] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [running]> on host lucas-Estudos
[2023-07-17 22:03:38,362] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=AAPL
AIRFLOW_CTX_EXECUTION_DATE=2023-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-30T00:00:00+00:00
[2023-07-17 22:03:47,184] {base.py:297} ERROR - AAPL: No price data found, symbol may be delisted (1h 2023-05-29 -> 2023-05-30)
[2023-07-17 22:03:47,189] {python.py:173} INFO - Done. Returned value was: None
[2023-07-17 22:03:47,246] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=AAPL, execution_date=20230530T000000, start_date=20230718T010338, end_date=20230718T010347
[2023-07-17 22:03:47,311] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-17 22:03:47,452] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-18 10:43:19,553] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-18 10:43:19,581] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [queued]>
[2023-07-18 10:43:19,582] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-07-18 10:43:19,582] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-07-18 10:43:19,582] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-07-18 10:43:19,612] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): AAPL> on 2023-05-30 00:00:00+00:00
[2023-07-18 10:43:19,623] {standard_task_runner.py:52} INFO - Started process 16320 to run task
[2023-07-18 10:43:19,630] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'AAPL', 'scheduled__2023-05-30T00:00:00+00:00', '--job-id', '1356', '--raw', '--subdir', 'DAGS_FOLDER/get_stocks.py', '--cfg-path', '/tmp/tmphk69n51d', '--error-file', '/tmp/tmpragvhzb5']
[2023-07-18 10:43:19,632] {standard_task_runner.py:80} INFO - Job 1356: Subtask AAPL
[2023-07-18 10:43:19,839] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.AAPL scheduled__2023-05-30T00:00:00+00:00 [running]> on host lucas-Estudos
[2023-07-18 10:43:20,052] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=AAPL
AIRFLOW_CTX_EXECUTION_DATE=2023-05-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-30T00:00:00+00:00
[2023-07-18 10:43:20,705] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/lucas/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/lucas/Área de Trabalho/Alura/EngenhariaDados/3-Executores do Airflow/Local and Celery/airflow_pipeline/dags/get_stocks.py", line 19, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/lucas/.local/lib/python3.10/site-packages/yfinance/utils.py", line 105, in wrapper
    result = func(*args, **kwargs)
  File "/home/lucas/.local/lib/python3.10/site-packages/yfinance/base.py", line 299, in history
    logger.debug(f'{self.ticker}: yfinance received OHLC data: {quotes.index[0]} -> {quotes.index[-1]}')
  File "/home/lucas/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 5175, in __getitem__
    return getitem(key)
  File "/home/lucas/.local/lib/python3.10/site-packages/pandas/core/arrays/datetimelike.py", line 370, in __getitem__
    "Union[DatetimeLikeArrayT, DTScalarOrNaT]", super().__getitem__(key)
  File "/home/lucas/.local/lib/python3.10/site-packages/pandas/core/arrays/_mixins.py", line 272, in __getitem__
    result = self._ndarray[key]
IndexError: index 0 is out of bounds for axis 0 with size 0
[2023-07-18 10:43:20,749] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=AAPL, execution_date=20230530T000000, start_date=20230718T134319, end_date=20230718T134320
[2023-07-18 10:43:20,771] {standard_task_runner.py:92} ERROR - Failed to execute job 1356 for task AAPL (index 0 is out of bounds for axis 0 with size 0; 16320)
[2023-07-18 10:43:20,806] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-07-18 10:43:20,908] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
